content/ct3a-guestbook.mdx

```diff
@@ -496,17 +538,17 @@ For this we can use optimistic UI updates! `react-query` makes this trivial to d
 
 ```ts:src/pages/index.tsx
 const ctx = trpc.useContext();
```

Comment:
utils is the conventional variable name for the trpc 10 context

Reply:
oh i didn't know this, why tho? doesn't ctx refer to context?